packages/cypher-builder/src/expressions/operations/math.ts

```diff
@@ -67,3 +66,39 @@ export function plus(...exprs: Expr[]): MathOp {
 export function minus(leftExpr: Expr, rightExpr: Expr): MathOp {
     return createOp("-", [leftExpr, rightExpr]);
 }
+
+/**
+ * @see [Cypher Documentation](https://neo4j.com/docs/cypher-manual/current/syntax/operators/#query-operators-mathematical)
+ * @group Expressions
+ * @category Operators
+ */
+export function multiply(leftExpr: Expr, rightExpr: Expr): MathOp {
+    return createOp("*", [leftExpr, rightExpr]);
+}
+
+/**
+ * @see [Cypher Documentation](https://neo4j.com/docs/cypher-manual/current/syntax/operators/#query-operators-mathematical)
+ * @group Expressions
+ * @category Operators
+ */
+export function divide(leftExpr: Expr, rightExpr: Expr): MathOp {
+    return createOp("/", [leftExpr, rightExpr]);
+}
+
+/**
+ * @see [Cypher Documentation](https://neo4j.com/docs/cypher-manual/current/syntax/operators/#query-operators-mathematical)
+ * @group Expressions
+ * @category Operators
+ */
+export function remainder(leftExpr: Expr, rightExpr: Expr): MathOp {
```

Comment:
According to the documentation this seems to be called `modulo` in Cypher, so maybe we could go with `mod`